### ISSUE TITLE
there is no constant name MAXIMUM_FRESH_DURATION_SECONDS

### ISSUE THREAD

**Issue by sanjsingh07** [overmind-xyz/oracle-of-delphi#1]
" /*
Returns the latest price as long as it is attested within the default maximum attestation
duration. The default maximum attestation duration is defined by the
MAXIMUM_FRESH_DURATION_SECONDS constant. Abort if the price is stale or if the pair does
not exist.
@param pair - id of the coin pair being updated
@return - the price of the pair along with the confidence
*/
public fun get_price(pair: String): Price acquires PriceBoard {
" while the MAXIMUM_FRESH_DURATION_SECONDS constant is nowhere to be found, and because of it I didn't put up the stale price check and now I'm unable to pass the last test, so tell me how should I proceed with this?

also already created this issue in a private repo assigned to me, creating this one here just so we can be consistent.

![image](https://github.com/overmind-xyz/oracle-of-delphi/assets/36148196/24268b09-b084-476a-9d75-ce6a229f079a)

**Comment by Annongkhanh** [comment]
Default MAXIMUM_FRESH_DURATION_SECONDS is 3 hours.  I found it while reading the description.

**Comment by sanjsingh07** [comment]
yup! I'm aware of it, I created this ticket so I can create a pull request and point it to this issue for further explanation.
